### PR TITLE
Add a demo for ptp grandmaster and slave

### PR DESCRIPTION
**Pull request by SchSeba**
Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

File: feature-configs/demo/ptp/ptpconfig-grandmaster.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+apiVersion: ptp.openshift.io/v1
```
Comment (fedepaol) [inline feature-configs/demo/ptp/ptpconfig-grandmaster.yaml]: Would it make sense to provide a default for these manifests and only the kustomizations under demo?

File: feature-configs/demo/ptp/ptpconfig-slave.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+apiVersion: ptp.openshift.io/v1
```
Comment (fedepaol) [inline feature-configs/demo/ptp/ptpconfig-slave.yaml]: Same for this file

File: feature-configs/demo/ptp/ptpconfig-grandmaster.yaml (hunk 1 above)
Comment (SchSeba) [reply feature-configs/demo/ptp/ptpconfig-grandmaster.yaml]: this is not a production way you should not configure the master to be one of the nodes.
All the nodes in the cluster should only be slaves, we do it like that in our environment for testing (we don't have an external device to sync from) this is why this file is in the demo folder

**Comment by SchSeba** [comment]
@fedepaol thanks for the comment I was thinking about that but I will like to leave the base folder to be the installation process only what you think?

File: feature-configs/demo/ptp/ptpconfig-grandmaster.yaml (hunk 1 above)
Comment (fedepaol) [reply feature-configs/demo/ptp/ptpconfig-grandmaster.yaml]: Ah, got it. Makes sense, thanks for the explanation

File: feature-configs/demo/ptp/ptpconfig-slave.yaml (hunk 2 above)
Comment (fedepaol) [reply feature-configs/demo/ptp/ptpconfig-slave.yaml]: This should belong to base then?

**Comment by fedepaol** [comment]
> @fedepaol thanks for the comment I was thinking about that but I will like to leave the base folder to be the installation process only what you think?

Yes, but having the slaves set is part of the installation of a cluster I think.
You should see the base as how would you setup a "cluster version" so you cna change as less as you can for the different cluster flavours.
In that sense, I would have the slaves configuration as part of base, customizing the interface name or something like that.

**Comment by SchSeba** [comment]
@fedepaol can you take another look?

File: feature-configs/demo/ptp/ptpconfig-slave.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+kind: PtpConfig
```
Comment (fedepaol) [inline feature-configs/demo/ptp/ptpconfig-slave.yaml]: Now that the slave profile is in base, can we remove this (or maybe just have this as a little patch?)

**Pull request merged**